### ISSUE TITLE
Password local configuration

### ISSUE THREAD

**Issue by jonasleu** [SafeExamBrowser/seb-win-refactoring#277]
**IMPORTANT**
Please _always_ consult the documentation first before creating a bug report!
https://safeexambrowser.org/windows/win_usermanual_en.html

**Describe the Bug**
The teacher can no longer get into the Safe Exam Browser Configuration Tool. It always comes, enter the password for the local configuration

**Steps to Reproduce**
1. The teacher bought a new laptop and he cannot longer work with the SEB
2. If he wants to create an exam or login into an existing one, he cannot enter, because he doesn't know the password for the local configuration. The passwords he used the past two years, doesn't work anymore! 

**Expected Behavior**
Login as before!

**Comment by jonasleu** [comment]
He uses the SEB 3.3.1. and Windows 10 Education Version 21H1

**Comment by dbuechel** [comment]
In this case, you can simply delete the local client configuration(s). Their specific location(s) and purpose are described under https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane:

```
%PROGRAMDATA%\SafeExamBrowser\SebClientSettings.seb
%APPDATA%\SafeExamBrowser\SebClientSettings.seb
```

**Comment by jonasleu** [comment]
Hi @dbuechel , 

I have a question.
How can I turn off the auto-correction for free text answers for students solving the exam on Mac? 

Thanks for your help, 
Jonas

**Comment by dbuechel** [comment]
There is a spell-checking feature in SEB for macOS / iOS (see https://safeexambrowser.org/macosx/mac_usermanual_en.html#AppearancePane), so that might be what you're looking for but I'd ask this question in the [repository for macOS / iOS](https://github.com/SafeExamBrowser/seb-mac/discussions) to be sure. Also, please always create a new discussion or issue for new question which is not related to the original question or issue.

**Comment by danschlet** [comment]
There was a bug relating auto-correction for the Moodle rich text editor (doesn't happen with the plain text editor). It's fixed in the [SEB 3.1 release candidate](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.1rc), if you want to test it.

**Comment by Irohpeter** [comment]
Every time I try to open SEB 3.9, on my windows 10 pro, it keeps asking password for local client registration.

how can I resolve this
